tests/manage/mcg/test_write_to_bucket.py

```diff
@@ -70,10 +70,68 @@ def test_write_file_to_bucket(self, mcg_obj, awscli_pod, bucket_factory):
     @pytest.mark.polarion_id("OCS-1949")
     @tier1
     @acceptance
-    def test_data_reduction(self, mcg_obj, awscli_pod, bucket_factory):
+    def test_data_deduplication(self, mcg_obj, awscli_pod, bucket_factory):
+        """
+        Test data deduplication mechanics
+        Args:
+            mcg_obj (obj): An object representing the current state of the MCG in the cluster
+            awscli_pod (pod): A pod running the AWSCLI tools
+            bucket_factory: Calling this fixture creates a new bucket(s)
+        """
+        download_dir = '/aws/deduplication/'
+        awscli_pod.exec_cmd_on_pod(
+            command=craft_s3_command(
+                f'cp s3://{constants.TEST_FILES_BUCKET}/danny.webm {download_dir}danny.webm'
+            ),
+            out_yaml_format=False
+        )
+        bucket = bucket_factory(amount=1)[0]
+        bucketname = bucket.name
+        for i in range(1, 3):
+            awscli_pod.exec_cmd_on_pod(
+                command=craft_s3_command(
+                    f'cp f"s3://{bucketname}/danny{i}.webm {download_dir}danny.webm'
+                ),
+                out_yaml_format=False
+            )
+        assert mcg_obj.check_data_reduction(bucketname, 43 * 1024 * 1024), (
+            'Data deduplication did not work as anticipated.'
+        )
+
+    @pytest.mark.polarion_id("OCS-1949")
+    @tier1
+    @acceptance
+    def test_data_compression_functionality(self, mcg_obj, awscli_pod, bucket_factory):
         """
         Test data reduction mechanics
+        Args:
+            mcg_obj (obj): An object representing the current state of the MCG in the cluster
+            awscli_pod (pod): A pod running the AWSCLI tools
+            bucket_factory: Calling this fixture creates a new bucket(s)
+        """
+        download_dir = '/aws/reduction/'
+        awscli_pod.exec_cmd_on_pod(
+            command=craft_s3_command(
+                f'cp s3://{constants.TEST_FILES_BUCKET}/enwik8 {download_dir}'
+            ),
+            out_yaml_format=False
+        )
+        bucket = bucket_factory(amount=1)[0]
+        bucketname = bucket.name
+        full_object_path = f"s3://{bucketname}"
+        sync_object_directory(
+            awscli_pod, download_dir, full_object_path, mcg_obj
+        )
+        assert mcg_obj.check_data_reduction(bucketname, 35 * 1024 * 1024), (
+            'Data compression did not work as anticipated.'
+        )
 
+    @pytest.mark.polarion_id("OCS-1949")
+    @tier2
+    @performance
+    def test_data_reduction_performance(self, mcg_obj, awscli_pod, bucket_factory):
```

Comment:
```suggestion
    def test_mcg_data_reduction_performance(self, mcg_obj, awscli_pod, bucket_factory):
```

Reply:
Let's skip the test for now with a marker that says it's inconsistent and leads to inconsistent results at the moment